### ISSUE TITLE
[Feature] : Want to add ContactUs page in website

### ISSUE THREAD

**Issue by Jivan052** [PratikMane0112/TelMedSphere#19]
### Description

![Screenshot 2024-12-25 014249](https://github.com/user-attachments/assets/3a042176-5f8c-4464-b222-4810b7e56005)
The website's navbar currently only includes a Login option, but it lacks essential sections like Contact Us, About Us, and FAQs. A well-designed navbar is crucial for easy navigation and aligns with industry standards for most websites, ensuring a better user experience.

Additionally, it is important to create pages such as Contact Us, About Us, and FAQs with an appealing and modern UI to enhance the overall look and functionality of the website.

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/PratikMane0112/TelMedSphere/blob/master/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [X] I am a SWOC'25 contributor.
- [ ] I am a KWOC'24 contributor.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/PratikMane0112/TelMedSphere/blob/master/.github/CODE_OF_CONDUCT.md)".

**Comment by Jivan052** [comment]
@PratikMane0112 assign this issue to me

**Comment by PratikMane0112** [comment]
Hey @Jivan052  Welcome to 💖TelMedSphere !!!!!!! Thank you for your interest but you just get this issue in wrong way. After login only the options are visible in navbar. You can check also it, try login with test accounts :
For patient -
username - patient@gmail.com
password - patient@gmail.com
For doctor -
username - doctor@gmail.com
password - doctor@gmail.com

Note : Do not forget to logout with accounts, when are done ....

**Comment by PratikMane0112** [comment]
Hey @Jivan052 Welcome to 💖TelMedSphere !!!!!!! Thank you for your interest but you just get this issue in wrong way. After login only the options are visible in navbar. You can check also it, try login with test accounts :
For patient -
username - [patient@gmail.com](mailto:patient@gmail.com)
password - [patient@gmail.com](mailto:patient@gmail.com)
For doctor -
username - [doctor@gmail.com](mailto:doctor@gmail.com)
password - [doctor@gmail.com](mailto:doctor@gmail.com)

Note : Do not forget to logout with accounts, when are done ....

**Comment by PratikMane0112** [comment]
Hey @Jivan052, are you reporting same issue as in #20 for contact us

**Comment by Jivan052** [comment]
No

**Comment by PratikMane0112** [comment]
Can you please tell me about your issue more ?

**Comment by Jivan052** [comment]
Yes, as you mentioned, after logging in, features like Model, Doctors, Medicines, and Home are displayed. However, since the About Us, Contact Us, and FAQs sections are located in the footer and their pages are not yet ready, I propose focusing on developing the About Us, Contact Us, and FAQs pages.

**Comment by PratikMane0112** [comment]
Yes, I understood your issue, please go ahead. Work on only Contact Us page, as for About Us Page it has been already assign to other in #20 .

**Comment by PratikMane0112** [comment]
Note : You can start working on issue but as per SWoC guidelines, I can merge your PR only after 1 Jan Onwards.

**Comment by PratikMane0112** [comment]
Also for any query join our discord server, link in README file

**Comment by Jivan052** [comment]
Sure

**Comment by PratikMane0112** [comment]
Hey @Jivan052, please complete it in two days, else it will vet closed
